### PR TITLE
app-text/master-pdf-editor: version bump to 5.3.22

### PR DESCRIPTION
**Pull request by jpataias**
version bump to 5.3.22

### CODE REVIEW

File: app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+RDEPEND="
+	>=dev-qt/qtcore-5.4.1:5
```
Comment (monsieurp) [inline app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild]: What you should you instead is make sure the package depends on Qt version 5. Change these lines to:
```
dev-qt/qtcore:5=
```
You should also change the lines below related to Qt. Make sure to read https://devmanual.gentoo.org/general-concepts/dependencies/ if you want to find out more about slots and subslots.

File: app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	>=dev-qt/qtgui-5.4.1:5
+	>=dev-qt/qtnetwork-5.4.1:5
+	>=dev-qt/qtprintsupport-5.4.1:5
+	>=dev-qt/qtsvg-5.4.1:5
+	>=media-gfx/sane-backends-1.0
```
Comment (monsieurp) [inline app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild]: This is long gone as well. Change it to:
```
media-gfx/sane-backends
```

**Comment by steils** [comment]
It is 5.3.22 already...

File: app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild (hunk 1 above)
Comment (mgorny) [reply app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild]: No. Qt uses subslots for a different purpose.

Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild]: So, should I make the change or not?
The only reason that I have made it as such is that these are the versions specified in the software (same for the sane-backends).

Comment (mgorny) [reply app-text/master-pdf-editor/master-pdf-editor-5.3.20.ebuild]: Please just disregard the subslot comments.

**Comment by mgorny** [comment]
In my opinion, this is good enough. However, as @steils pointed out, it would be probably best to update it to the newest version first.

**Comment by jpataias** [comment]
Thanks, I've just updated the commit to the latest version.

**Pull request closed without merging**